sdk/canton/BUILD.bazel

```diff
@@ -39,11 +39,11 @@ cat << EOF > $@
 package com.digitalasset.canton.buildinfo
 
 case object BuildInfo {{
-  val version: String = "2.7.0-SNAPSHOT"
+  val version: String = "2.9.0-SNAPSHOT"
```

Comment:
Could we get that automatically ? (in any case not for this PR).

Reply:
It is supposed to happen with the monorepo..

Reply:
fun

Reply:
Since daml sdk version should match canton sdk version, we just need to use SDK version